app/src/main/java/com/example/rwothoromo/developers/view/MainActivity.java

```diff
@@ -24,24 +30,42 @@
 	Parcelable githubUserListState = null;
 	private RecyclerView recyclerView;
 	private RecyclerView.LayoutManager layoutManager;
+	private SwipeRefreshLayout swipeRefreshLayout;
 
+	@RequiresApi(api = Build.VERSION_CODES.KITKAT)
 	@Override
 	protected void onCreate(Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
 		setContentView(R.layout.activity_main);
 		Toolbar toolbar = findViewById(R.id.toolbar);
 		setSupportActionBar(toolbar);
+		Objects.requireNonNull(getSupportActionBar()).setSubtitle("Location: Nairobi, Stack: Java");
 
 		recyclerView = findViewById(R.id.recyclerView);
 		recyclerView.setHasFixedSize(true);
+		swipeRefreshLayout = findViewById(R.id.swiperefresh);
 
-		layoutManager = new LinearLayoutManager(this);
+		layoutManager = new GridLayoutManager(this, 1);
```

Comment:
Can you use a higher span count to make it actually display like a grid?

Reply:
I could update it to `two` and list two items per row.